src/autoyast-rnc/auth-client.rnc

```diff
@@ -1,301 +0,0 @@
-default namespace = "http://www.suse.com/1.0/yast2ns"
```

Comment:
> It defines elements that clearly violate XML, such as those beginning with letter 'l' and 'm'.

What? `<l>` or `<m>` is a perfectly valid XML element.

> There are way too many configuration keys that would be painful to be serialised into XML and hence the module exports/imports JSON instead.

UGH. Seriously? Yes, the number of configuration elements is large and it may be hard to maintain, so I see the desire to somehow reduce the maintenance of the schema. But why abandon XML? Please give an example.